code/game/gamemodes/game_mode.dm

```diff
@@ -124,159 +102,284 @@
+		dat += "[round_statistics.queen_screech] Queen screeches."
+	if(round_statistics.ravager_ravage_victims)
+		dat += "[round_statistics.ravager_ravage_victims] ravaged victims. Damn, Ravagers!"
+	if(round_statistics.warrior_limb_rips)
+		dat += "[round_statistics.warrior_limb_rips] limbs ripped off by Warriors."
+	if(round_statistics.crusher_stomp_victims)
+		dat += "[round_statistics.crusher_stomp_victims] people stomped by crushers."
+	if(round_statistics.praetorian_spray_direct_hits)
+		dat += "[round_statistics.praetorian_spray_direct_hits] people hit directly by Praetorian acid spray."
+	if(round_statistics.weeds_planted)
+		dat += "[round_statistics.weeds_planted] weed nodes planted."
+	if(round_statistics.weeds_destroyed)
+		dat += "[round_statistics.weeds_destroyed] weed tiles removed."
+	if(round_statistics.carrier_traps)
+		dat += "[round_statistics.carrier_traps] hidey holes for huggers were made."
+	if(round_statistics.sentinel_neurotoxin_stings)
+		dat += "[round_statistics.sentinel_neurotoxin_stings] number of times Sentinels stung."
+	if(round_statistics.drone_stings)
+		dat += "[round_statistics.drone_stings] number of times Drones stung."
+	if(round_statistics.drone_salvage_plasma)
+		dat += "[round_statistics.drone_salvage_plasma] number of times Drones salvaged corpses."
+	if(round_statistics.defiler_defiler_stings)
+		dat += "[round_statistics.defiler_defiler_stings] number of times Defilers stung."
+	if(round_statistics.defiler_neurogas_uses)
+		dat += "[round_statistics.defiler_neurogas_uses] number of times Defilers vented neurogas."
+	var/output = jointext(dat, "<br>")
+	for(var/mob/player in GLOB.player_list)
+		if(player?.client?.prefs?.toggles_chat & CHAT_STATISTICS)
+			to_chat(player, output)
+
+
+/datum/game_mode/proc/count_humans_and_xenos(list/z_levels = SSmapping.levels_by_any_trait(list(ZTRAIT_MARINE_MAIN_SHIP, ZTRAIT_LOW_ORBIT, ZTRAIT_GROUND)))
+	var/num_humans = 0
+	var/num_xenos = 0
+
+	for(var/mob/M in GLOB.player_list)
+		if(M.z && (M.z in z_levels) && M.stat != DEAD && !isspaceturf(M.loc)) //If they have a z var, they are on a turf.
+			if(ishuman(M) && !(M.status_flags & XENO_HOST) && !iszombie(M))
+				var/mob/living/carbon/human/H = M
+				if(H.species && H.species.count_human) //only real humans count
+					num_humans++
+			else if(isxeno(M))
+				var/mob/living/carbon/Xenomorph/X = M
+				if(!X.stealth_router(HANDLE_STEALTH_CHECK)) //We don't count stealthed Beanos due to delay potential
+					num_xenos++
+
+	return list(num_humans,num_xenos)
+
+
+/datum/game_mode/proc/disperse_fog()
+	set waitfor = FALSE
+
+	flags_round_type &= ~MODE_FOG_ACTIVATED
+	var/i
+	for(i in GLOB.fog_blockers)
+		qdel(i)
+		sleep(1)
+
+
+/datum/game_mode/proc/attempt_to_join_as_larva(mob/xeno_candidate)
+	to_chat(xeno_candidate, "<span class='warning'>This is not available in this gamemode.</span>")
+	return FALSE
+
+
+/datum/game_mode/proc/spawn_larva(mob/xeno_candidate, var/mob/living/carbon/Xenomorph/Queen/mother)
```

Comment:
These procs should work in extended, for events.
Perhaps `/datum/game_mode/xenomorph/distress_signal` with a `/datum/game_mode/xenomorph/extended` option, and the same for a future WO.
Or just make these for all game modes.